rock_paper_scissors/exc.py

```diff
@@ -0,0 +1,49 @@
+class ProgramError(Exception):
+    """Base exception class for errors raised by rock paper scissors program."""
+    pass
+
+
+class UserDefinedError(ProgramError):
+    """Raised for all user input errors."""
+    pass
+
+
+class UserChoiceInputError(UserDefinedError):
+    """Raised for invalid User sting input."""
+    def __init__(self, message):
+        self.message = message
+
+
+class UserRoundsInputError(UserDefinedError):
+    """Raised for invalid rounds input."""
+    pass
+
+
+class RoundsStringError(UserRoundsInputError):
+    """Raised if rounds input is a string."""
+    def __init__(self, message):
```

Comment:
This part has been catching me up. I will reach out today for a better understanding of what elements I may need for the Exception classes.